src/debug-session/gdbtarget-debug-session.ts

```diff
@@ -63,12 +63,12 @@ export class GDBTargetDebugSession {
                 context: 'hover'
             };
             const response = await this.session.customRequest('evaluate', args) as DebugProtocol.EvaluateResponse['body'];
-            return response.result.match(/\d+/) ? response.result : undefined;
```

Comment:
I can't see the `match` being moved to the so far only caller in cpu-states. This is even more necessary now that errors are printed to the return string. cpu-states expects a number that is used for calculations from cycles to time.